Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:18-alpine AS build
+WORKDIR /app
+COPY angular/package*.json ./
+RUN npm ci
+COPY . .
+WORKDIR /app/angular
+RUN npm run build
+
+FROM nginx:alpine
```

Comment:
you version should always PINNED to SHA digest of the image to avoid building incompatible version in the future.